src/bioversions/sources/rxnorm.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+"""A getter for RxNorm."""
+
+from ..utils import Getter, VersionType, get_soup
+from datetime import datetime
+
+__all__ = [
+    "RxNormGetter",
+]
+
+URL = "https://www.nlm.nih.gov/research/umls/rxnorm/docs/rxnormfiles.html"
+
+
+class RxNormGetter(Getter):
+    """A getter for RxNorm."""
+
+    bioregistry_id = "rxnorm"
```

Comment:
RxNorm is unfortunately not registered with identifiers.org or the BioRegistry -- I assumed it's tentatively fine to supply a reasonable value here?

Reply:
http://bioregistry.io/registry/rxnorm should be what you're looking for

Reply:
Ah, great -- not sure how I missed that... very easily-findable entry.